op-chain-ops/genesis/layer_one.go

```diff
@@ -61,43 +58,52 @@ func BuildL1DeveloperGenesis(config *DeployConfig, dump *ForgeAllocs, l1Deployme
 	}
 	// copy, for safety when the dump is reused (like in e2e testing)
 	genesis.Alloc = dump.Copy().Accounts
-	memDB := state.NewMemoryStateDB(genesis)
-	FundDevAccounts(memDB)
-	SetPrecompileBalances(memDB)
+	FundDevAccounts(genesis)
+	SetPrecompileBalances(genesis)
 
 	l1Deployments.ForEach(func(name string, addr common.Address) {
-		acc := memDB.GetAccount(addr)
-		if acc != nil {
-			log.Info("Included L1 deployment", "name", name, "address", addr, "balance", acc.Balance, "storage", len(acc.Storage), "nonce", acc.Nonce)
-		} else {
+		acc := genesis.Alloc[addr]
+		if isAccountEmpty(acc) {
```

Comment:
```suggestion
		acc, ok := genesis.Alloc[addr]
		if !ok {
```

Empty accounts shouldn't be included in the `Alloc` right, so can't we just check the entry presence, rather than loading a default account value and checking it's all zeroed?

Reply:
ah this is far better